test/smokeTest.js

```diff
@@ -147,13 +147,13 @@ test("smoke tests", assert => {
         ])
       return updateSrcP
         .then(() => {
-          assert.equals(browser.window._noAcceptReloaded, true)
+          assert.equals(browser.window._noAcceptReloaded, false)
```

Comment:
Are you saying the naming is confusing now? I agree, but not sure what a better name is. I tried changing the locations of `true` and `false` and I think it makes more sense now.